types/seat/wlr_seat.c

```diff
@@ -281,6 +284,8 @@ struct wlr_seat *wlr_seat_create(struct wl_display *display, const char *name) {
 	wl_signal_init(&seat->events.touch_grab_begin);
 	wl_signal_init(&seat->events.touch_grab_end);
 
+	wl_signal_init(&seat->events.focus_change);
```

Comment:
What kind of focus change? There are several.

Reply:
The kind of focus change where the focused surface and client change.